# msg_mapDiff
msg_KeyFrame[] KeyFramesToAddorUpdate
msg_MapPoint[] MapPointsToAddorUpdate
msg_Measurement[] MeasurementsToAddorUpdate
int64[] KeyFramesToDelete
int64[] MapPointsToDelete
msg_Meas_id[] MeasurementsToDelete